packages/gp2-components/src/utils/icon.ts

```diff
@@ -0,0 +1,37 @@
+import { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import {
+  outputArticle,
+  outputCode,
+  outputDataset,
+  outputForm,
+  outputMaterial,
+  outputReport,
+  projectIcon,
+  workingGroupIcon,
+} from '../icons';
+
+export const getIconForDocumentType = (
```

Comment:
This function also exists here, maybe we should refactor to have only one:

https://github.com/yldio/asap-hub/blob/813c79f09256a152ee3d37f46bbff0094c8be387/packages/gp2-components/src/organisms/OutputRelatedResearchCard.tsx#L17-L36

Reply:
I've done that here https://github.com/yldio/asap-hub/pull/3779/files#diff-404baf71a79662d27a9e9b4f2ef852c3bd91aeb57a97938be56ad04c6eee2408 but this is still in PO so it hasn't been merged yet